payments/vouchers.go

```diff
@@ -27,10 +28,15 @@ type Voucher struct {
 	Signature state.Signature
 }
 
-// Balance stores the remaining and paid funds in a channel.
-type Balance struct {
-	Remaining *big.Int
-	Paid      *big.Int
+// VoucherInfo contains the largest voucher we've received on a channel.
+// As well as details about the balance and who the payee/payer is.
+type VoucherInfo struct {
```

Comment:
This was based on the `paymentStatus` we were storing in maps in the voucher manager.